tt_metal/distributed/mesh_device.hpp

```diff
@@ -79,7 +83,7 @@ class MeshDevice : public std::enable_shared_from_this<MeshDevice> {
     MeshDeviceID mesh_id;
     MeshShape mesh_device_shape;
     MeshType type;
-    std::shared_ptr<MeshDeviceView> primary_view;
+    std::unique_ptr<MeshDeviceView> primary_view;
```

Comment:
Just wondering - is there non-primary view?

Reply:
Also great switch to a unique_ptr here!

Reply:
At some point, but not anymore -- I've renamed it so it's clearer.